src/store/audio.ts

```diff
@@ -2060,17 +2063,84 @@ export const audioCommandStore = transformCommandStore(
             );
 
             // 読みの内容が変わっていなければテキストだけ変更
-            const isChangedMora = isAccentPhrasesTextDifferent(
+            const isSameText = !isAccentPhrasesTextDifferent(
               query.accentPhrases,
               accentPhrases
             );
+            let newAccentPhrases: AccentPhrase[] = [];
+            if (isSameText) {
+              newAccentPhrases = accentPhrases;
```

Comment:
元実装的には、こう･･･？
```suggestion
              newAccentPhrases = query.accentPhrases;
```

･･････テストあると嬉しいな～～！！
（プルリクエストの範囲外ですが････。）